### ISSUE TITLE
Support server hot-reloading

### ISSUE THREAD

**Issue by l1b3r** [evolv-ai/serverless-offline-edge-lambda#113]
TL;DR: As of version `1.0.2` the plugin seems to be missing a hot-reload feature.

I am developing a TypeScript-based L@E function. My `serverless.yml` is configured this way:

```yaml
plugins:
  - serverless-offline-edge-lambda
  - serverless-plugin-typescript
custom:
   offlineEdgeLambda:
    path: .build
    originMap:
      - pathPattern: '*'
        target: dev_origins
. . .
```

When I start the server using `sls offline start --port 80 --fileDir ./dev_origins` I can successfully access my lambda at `localhost:80` and everything works as expected.

However, when I change the source code of the lambda in TS, the server doesn't refresh with the latest version.
On every change I can see the log from the `serverless-plugin-typescript`:
```
Serverless: Compiling with Typescript...
Serverless: Typescript compiled.
```
I can also confirm that the compiled JS in `.build` (which I have specified as a `path`) is also updated.

Can the hot-reload feature be implemented, so that when `Serverless: Typescript compiled.` comes up, the server reloads with the freshly compiled version?
